ci/lint.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+)
+
+const (
+	golangCiLintImage = "docker.io/golangci/golangci-lint@sha256:b5f8712114561f1e2fbe74d04ed07ddfd992768705033a6251f3c7b848eac38e"
```

Comment:
nit: we should also include the tag this in here so it's clear which version this is: `docker.io/golangci/golangci-lint:<version>@sha256:<hash>`